oteapi/models/transformationconfig.py

```diff
@@ -39,40 +39,58 @@ class TransformationConfig(GenericConfig, SecretConfig):
         description=(
             "Type of registered transformation strategy. E.g., `celery/remote`."
         ),
+        IRI="http://purl.org/dc/terms/type",  # type: ignore
     )
     name: Optional[str] = Field(
-        None, description="Human-readable name of the transformation strategy."
+        None,
+        description="Human-readable name of the transformation strategy.",
+        IRI="http://purl.org/dc/elements/1.1/title",  # type: ignore
```

Comment:
Better to use the widely adopted `dcterms` namespace, rather than the old and more limited `http://purl.org/dc/elements/1.1/` namespace.
```suggestion
        IRI="http://purl.org/dc/terms/title",  # type: ignore
```